services.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+
+services:
+  base:
+    image: mihaibivol/python-base
```

Comment:
I added you to the inspirehep dockerhub organization, so can you push these there?